src/main/java/com/ea/restaurant/entities/User.java

```diff
@@ -0,0 +1,16 @@
+package com.ea.restaurant.entities;
+
+import lombok.*;
+
+@EqualsAndHashCode(callSuper = false)
```

Comment:
This should be true, because user extendes from base entity and base entity contains the id

Reply:
Done.